/home/portal/anaconda3/envs/myenv/lib/python3.10/site-packages/distributed/node.py:182: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 41103 instead
  warnings.warn(
free(): invalid pointer
2024-10-25 00:05:31,720 - distributed.nanny - WARNING - Restarting worker

real	19m33.674s
user	0m0.000s
sys	0m0.028s
